backend/src/main/java/de/neuefische/backend/model/settingsSubModel/result/UserTimeVote.java

```diff
@@ -0,0 +1,16 @@
+package de.neuefische.backend.model.settingsSubModel.result;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+
+public class UserTimeVote {
```

Comment:
What is this model for? Do you use it anywhere? Names are not realy clear

Reply:
If I got time in the upcoming days, I will revise the different names. Thanks for your helpful remark!